packages/clay-charts/src/ChartBase.js

```diff
@@ -103,6 +103,10 @@ const ChartBase = {
 			return;
 		}
 
+		if (this._pollingInterval) {
+			clearInterval(this._pollingInterval);
+		}
```

Comment:
This should go to DataComponent, as ChartBase has no knowledge at all about polling

Reply:
Ok, but `ChartBase` (or `Chart` if you prefer) will have to call this method defined in `DataComponent` since it can't be called `disposed`. Do you agree?